index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EmpowerHer | Aim for Power</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/jgd5och.css">
+    <script src="https://kit.fontawesome.com/4dbfbae31e.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+        <div>
+            <i class="fa-sharp fa-solid fa-bars fa-2xl menu-icon" style="color: #545454; padding: 2rem 1.5rem;"></i>
+        </div>
+        <section class="header-one">
+            <div class="header-one-container">
+                <ul>
+                    <li><i class="fa-brands fa-facebook-f" style="color: #ffffff;"></i></li>
+                    <li><i class="fa-brands fa-twitter" style="color: #ffffff;"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </section>
+        <section class="header-two">
+            <div class="header-two-container">
+                <a href="./index.html"><img src="./assets/images/Color logo - no background.png"
+                        alt="EmpowerHer icon"></a>
+                <nav>
+                    <ul>
+                        <li><a href="./pages/about.html" class="tag">About</a></li>
+                        <li><a href="./pages/about.html">Program</a></li>
+                        <li><a href="./pages/about.html">Join</a></li>
+                        <li><a href="./pages/about.html">Sponsor</a></li>
+                        <li><a href="./pages/about.html">News</a></li>
+                        <li><a href="./pages/about.html" class="sponsor tag">Sponsor</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+        <section class="mobile-menu">
+            <i class="fa fa-close" style="color:#ffffff;"></i>
+            <ul>
+                <li><a href="./pages/about.html" class="tag">About</a></li>
+                <li><a href="./pages/about.html">Program</a></li>
+                <li><a href="./pages/about.html">Join</a></li>
+                <li><a href="./pages/about.html">Sponsor</a></li>
+                <li><a href="./pages/about.html">News</a></li>
+                <li><a href="./pages/about.html" class="sponsor tag">Sponsor</a></li>
+            </ul>
+        </section>
+    </header>
+    <main>
+        <section class="hero-section">
+            <article class="hero-article">
+                <h1>EmpowerHer Summit 2023:</h1>
+                <h2>Building the level ground for growth</h2>
+                <p>Join us for an empowering and transformative experience at EmpowerHer Summit 2023!
```

Comment:
- [ ]  Kindly make sure you stick to the original design as much as possible. your typo(fonts size, family ...) and the colors are supposed to match the original design.
here' are the colors you might consider using.
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/fcd66cda-db4a-48e0-a044-901b1f921d5b)

### original design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/dcb038e6-b098-4638-8b32-7a73af9788e3)
### Your design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/87d011f4-71a4-4541-92fb-70e34803f8e4)

Reply:
Thank you for bringing this to my notice @christianbiring1. I would like some clarification. Do you also mean the color of the header that says "Building the level ground for growth'? The header in the original has a background image I do not have access to and it might be difficult to get the same look as the original. If I am wrong, please correct me.

Reply:
hey @Wineshuga YOu may leave the image and make the fonts (size, family) identical